design-patterns/gc-intro/index.json-ld

```diff
@@ -0,0 +1,268 @@
+					}
+				],
+				"fr": [
+					{
+						"@type": "source-code",
+						"description": "Exemple de code :",
+						"code": "<div class=\"bg-center bg-cover\" data-bgimg-srcset=\"img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w\">\n\t<div class=\"container\">\n\t\t<div class=\"row mrgn-bttm-md\">\n\t\t\t<div class=\"col-md-7\">\n\t\t\t\t<h1 property=\"name\" id=\"wb-cont\">[Nom de l’institution]</h1>\n\t\t\t\t<p>Description courte du mandat de d’institution.</p>\n\t\t\t\t<p><a class=\"btn btn-call-to-action\" href=\"#\">Bouton de super-tâche</a></p>\n\t\t\t</div>\n\t\t</div>\n\t</div>\n</div>"
+					}
+				]
+			}
+		}
+	],
+	"iteration": [
+		{
+			"@id": "_:iteration_intro_1",
+			"name": "Introduction block - Iteration 1",
+			"date": "2024-03",
+			"detectableBy": "h1#wb-cont+p",
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": "<h1 property=\"name\" id=\"wb-cont\" dir=\"ltr\">Introduction block</h1>\n<p>The introduction block pattern introduces the content of a landing page.</p>\n<p><a class=\"btn btn-call-to-action\" href=\"#\">Super task button</a></p>"
+				}
+			]
+		},
+		{
+			"@id": "_:iteration_intro_image_1",
+			"name": "Introduction block with image - Iteration 1",
+			"date": "2024-03",
+			"detectableBy": ".bg-cover h1#wb-cont + p",
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": "<div class=\"bg-center bg-cover\" data-bgimg-srcset=\"img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w\">\n\t<div class=\"container\">\n\t\t<div class=\"row mrgn-bttm-md\">\n\t\t\t<div class=\"col-md-7\">\n\t\t\t\t<h1 property=\"name\" id=\"wb-cont\">[Institution name]</h1>\n\t\t\t\t<p>Short description of the institution’s mandate.</p>\n\t\t\t\t<p><a class=\"btn btn-call-to-action\" href=\"#\">Super task button [optional]</a></p>\n\t\t\t</div>\n\t\t</div>\n\t</div>\n</div>"
+				}
+			]
+		}
+	],
+	"changesets": [
+		{
+			"@id": "_:cs_intro",
+			"name": "Introduction block",
+			"status": "stable",
+			"baseOnIteration": "_:iteration_intro_1",
+			"detectableBy": "h1#wb-cont + p",
+			"semantic": "h1 + p",
+			"schema": [
+				"Page title",
+				"Page description",
+				"Page supertask (optional)"
+			],
+			"context": "The introduction block pattern is a mandatory element for institutional landing pages and topic pages. It’s optional for other landing pages."
+		},
+		{
+			"@id": "_:cs_intro_image",
+			"name": "Introduction block with image",
+			"status": "stable",
+			"baseOnIteration": "_:iteration_intro_image_1",
+			"detectableBy": ".bg-cover h1#wb-cont + p",
```

Comment:
Humm, this is not a component, it should be only suggestive, we should not record the arrangement.

Although, we can probably add some informal notes regarding the design pattern. That presentation should be different compared to other fully documented component.